version.txt

```diff
@@ -1 +1 @@
-1.1.5
+1.2.0
```

Comment:
Yeah - I also think this is a minor version bump, given that the end-user will experience a slight difference (a failure will be reported properly, rather than just swallowed with no supplied output)